lms/djangoapps/instructor/views/api.py

```diff
@@ -630,6 +630,8 @@ def students_update_enrollment(request, course_id):
     email_students = _get_boolean_param(request, 'email_students')
     is_white_label = CourseMode.is_white_label(course_id)
     reason = request.POST.get('reason')
+    role = request.POST.get('role')
+
     if is_white_label:
```

Comment:
additionally we should remove this `is_white_label` check and validate `reason` and `role` are present for manual enrollment since those fields are manadatory.